requirements/frozen.py2.txt

```diff
@@ -320,7 +320,7 @@ uritemplate==3.0.0
 urllib3==1.24.1
 virtualenv==16.4.3
 vspk==5.3.2
-wait-for==1.0.11
+wait-for==1.0.12
```

Comment:
Need to update the version in the docs frozen files as well.

Thanks for the enhancement!!